sumpy/expansion/m2l.py

```diff
@@ -173,7 +173,9 @@ def get_m2l_translation_class(
 
 # {{{ M2LTranslationBase
 
+# FIXME: these seem to be sym.Expr in every usage here. Any other cases?
 TranslationClassesDepData: TypeAlias = tuple[Any, ...]
```

Comment:
Safe to try out `tuple[sym.Expr, ...]`?

Reply:
If it typechecks, feel free to make it sharper.